ginivision/src/main/java/net/gini/android/vision/internal/ui/AlertDialogFragment.java

```diff
@@ -0,0 +1,107 @@
+package net.gini.android.vision.internal.ui;
+
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.StringRes;
+import android.support.v4.app.DialogFragment;
+import android.support.v4.app.FragmentActivity;
+import android.support.v7.app.AlertDialog;
+
+/**
+ * Created by Alpar Szotyori on 05.06.2018.
+ *
+ * Copyright (c) 2018 Gini GmbH.
+ */
+
+/**
+ * @exclude
+ */
+public class AlertDialogFragment extends DialogFragment {
+
+    private static final String ARG_TITLE = "ARG_TITLE";
+    private static final String ARG_MESSAGE = "ARG_MESSAGE";
+    private static final String ARG_POSITIVE_BUTTON_TITLE = "ARG_POSITIVE_BUTTON_TITLE";
+    private static final String ARG_NEGATIVE_BUTTON_TITLE = "ARG_NEGATIVE_BUTTON_TITLE";
+    private static final String ARG_DIALOG_ID = "ARG_DIALOG_ID";
+
+    @StringRes
+    private int mTitle;
+    private int mMessage;
+    private int mPositiveButtonTitle;
+    private int mNegativeButtonTitle;
+    private int mDialogId;
```

Comment:
Aren't `mMessage`, `mPositiveButtonTitle`, `mNegativeButtonTitle` and `mDialogId` a string resource as well?